core/src/main/java/org/elasticsearch/cluster/routing/OperationRouting.java

```diff
@@ -43,13 +45,24 @@
 
 public class OperationRouting extends AbstractComponent {
 
+    public static final Setting<Boolean> USE_ADAPTIVE_REPLICA_SELECTION_SETTING =
```

Comment:
if this is false we should not collect any statistics in the SearchTransportService either no?

Reply:
+1 to keep collecting